lib/blue_state_digital/error_middleware.rb

```diff
@@ -11,7 +12,9 @@ def on_complete(env)
       when 403
         raise BlueStateDigital::Unauthorized, response_values(env).to_s
       when 409
-        if env.body =~ /does not exist/
+        if env.body =~ /<error>/
+          raise BlueStateDigital::XmlErrorResponse, env.body
```

Comment:
When is the XML error sent on the response? Would it make sense to parse the XML to be clearer when reporting back to gem consumer what was the issue (in case it's easy to identify a few common error cases)?